src/Chapter08/Listing08.07.DeclarationWithoutContainingInterface.cs

```diff
@@ -1,41 +1,40 @@
-﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter08.Listing08_07
+﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter08.Listing08_07;
+
+interface IReadableSettingsProvider
+{
+    string GetSetting(string name, string defaultValue);
+}
+
+interface ISettingsProvider : IReadableSettingsProvider
 {
-    interface IReadableSettingsProvider
+
+    #region INCLUDE
+    #if COMPILEERROR // EXCLUDE
+    // ERROR:  GetSetting() not available on ISettingsProvider
+    string ISettingsProvider.GetSetting(
+        string name, string defaultValue)
     {
-        string GetSetting(string name, string defaultValue);
+        // ...
     }
+    #endif // COMPILEERROR // EXCLUDE
```

Comment:
```suggestion
    #endif // COMPILEERROR // EXCLUDE
```

Is it intentional to put COMPILEERROR in a comment?